### PR TITLE
Add Spring 2021 Fanart Contest Results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
🌹 

todo:

- [x] run script to generate seasonal backgrounds (show_in_client is set on all entries)
- [x] stable update
- [x] award profile badges
- [x] award supporter

### CODE REVIEW

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Spring 2021 Fanart Contest Results"
+date: 2021-05-20 08:00:00 +0000
+---
+
+You voted, and the results are in! Check out the winners behind our latest Seasonal backgrounds!
```
Comment (TPGPL) [inline news/2021-05-20-spring-fanart-contest-results.md:7]:
```suggestion
You voted, and the results are in! Check out the winners behind our latest seasonal backgrounds!
```
no need to capitalize i think?

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+[![](https://assets.ppy.sh/contests/125/banners/Sjao.jpg)](https://assets.ppy.sh/contests/125/winners/Sjao.png)
+
+[**Sjao**](https://osu.ppy.sh/users/7295733) returns again to snag **4437** votes with their utterly gorgeous take on a springtime pippi getting some reading done in a butterfly-laced park beneath a tree.
+
+Well known for their outstanding use of colour and light, Sjao continues their streak of standout performance among our fanart contests, and we couldn't be happier to welcome them back.
+
+[![](https://assets.ppy.sh/contests/125/banners/Porukana.jpg)](https://assets.ppy.sh/contests/125/winners/Porukana.png)
+
+[**Porukana**](https://osu.ppy.sh/users/12992775)'s interesting take on Mocha and pippi enjoying the spring rain wowed **4406** voters, and for very good reason!
```
Comment (TPGPL) [inline news/2021-05-20-spring-fanart-contest-results.md:51]:
```suggestion
[**Porukana**](https://osu.ppy.sh/users/12992775)'s interesting take on Mocha and pippi enjoying the spring rain wowed **4406** voters, and for a very good reason!
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+An extra special mention goes out to the creative use of theme here - to my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
+
+---
+
+[Download a zip file containing all the winning entries in high definition here!](https://assets.ppy.sh/contests/125/winners/SpringFanart2021-Winners.zip)
+
+Be sure to check out the [Spring 2021 Fanart Contest results](https://osu.ppy.sh/community/contests/125) for the full listing of all 120+ entries to get a better picture of the entire voting.
+
+The finalist pieces described above are now available as the Spring 2021 Seasonal backgrounds, the top 3 artists have earned themselves a swanky new profile badge (of their own work, no less!) and every finalist goes home with 3 months of osu!supporter for their efforts.
```
Comment (TPGPL) [inline news/2021-05-20-spring-fanart-contest-results.md:121]:
```suggestion
The finalist pieces described above are now available as the Spring 2021 seasonal backgrounds, the top 3 artists have earned themselves a swanky new profile badge (of their own work, no less!) and every finalist goes home with 3 months of osu!supporter for their efforts.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+A huge thank you to everyone who took the time to vote - we know these contests are never easy with the sometimes overwhelming conundrum of choices to be made in the face of so many amazing entries. Another big thank you to every artist who participated too - putting your work out there like this is not easy at all, and we hope you all had fun in the process.
```
Comment (TPGPL) [inline news/2021-05-20-spring-fanart-contest-results.md:123]:
```suggestion
A huge thank you to everyone who took their time to vote - we know these contests are never easy with the sometimes overwhelming conundrum of choices to be made in the face of so many amazing entries. Another big thank you to every artist who participated too - putting your work out there like this is not easy at all, and we hope you all had fun in the process.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+[![](https://assets.ppy.sh/contests/125/banners/PacifiedPaints.jpg)](https://assets.ppy.sh/contests/125/winners/PacifiedPaints.png)
+
+Continuing the trend of cute overload, [**PacifiedPaints**](https://osu.ppy.sh/users/8377453) followed up with **3749** votes as people could not help but appreciate their stunning take on a very fashionable pippi (and duck) chilling out in a spring sunshower.
+
+A certain colourless feline also makes a cameo appearance here as a logo on what we presume is a bespoke line of osu!-related clothing.
+
+[![](https://assets.ppy.sh/contests/125/banners/McFriedFries.jpg)](https://assets.ppy.sh/contests/125/winners/McFriedFries.png)
+
+Dappled light enchanted **3197** voters in [**McFriedFries**](https://osu.ppy.sh/users/12694139)'s picnic piece sporting pippi, Mocha and Chirou enjoying some time (and burgers) in the sun.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:81]: optimising filesize

```suggestion
Dappled light enchanted **3197** voters in [**McFriedFries**](https://osu.ppy.sh/users/12694139)' picnic piece sporting pippi, Mocha and Chirou enjoying some time (and burgers) in the sun.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+There's just so much going on in this piece that it is hard to praise all of it in only a few short paragraphs. Absolutely stunning, and more than deserving of its place in the finalists.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:83]: while yes, this article does vary contraction usage, i feel this fits better

```suggestion
There's just so much going on in this piece that it's hard to praise all of it in only a few short paragraphs. Absolutely stunning, and more than deserving of its place in the finalists.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+[![](https://assets.ppy.sh/contests/125/banners/Poring.jpg)](https://assets.ppy.sh/contests/125/winners/Poring.png)
+
+**3107** found themselves lost in [**Poring**](https://osu.ppy.sh/users/14743626)'s urban cityscape vista alongside a uniformed pippi in what we now take to be a still shot of the canonical osu! universe, complete with A Labour of Love billboards flying in the distance.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:87]: 3107 what

```suggestion
**3107** mesmerised individuals found themselves lost in [**Poring**](https://osu.ppy.sh/users/14743626)'s urban cityscape vista alongside a uniformed pippi in what we now take to be a still shot of the canonical osu! universe, complete with A Labour of Love billboards flying in the distance.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+I'll break from tradition here a little bit to mention that this was one of my personal favourites in the entire contest due to how wonderfully stylized, crisp and clean it is. Everything about it feels so delightfully *deliberate* in a way that is hard to describe but very easy to appreciate.
+
+[![](https://assets.ppy.sh/contests/125/banners/araran.jpg)](https://assets.ppy.sh/contests/125/winners/araran.png)
+
+**2767** voters succumbed to the charms of [**araran**](https://osu.ppy.sh/users/8937198)'s catgirl pippi set to a light-infused backdrop of vibrant sakura trees and clouds.
+
+With the one-two punch of gorgeous background and cute mascot all wrapped up in some serious artistic chops, it's no surprise that araran's first fanart entry ended with a spot in the finalists.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:95]:
```suggestion
With the one-two punch of gorgeous background and cute mascot all wrapped up in some serious artistic chops, it's no surprise that araran's first fanart entry ended with a spot among the finalists.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+[![](https://assets.ppy.sh/contests/125/banners/Jinairu[Sensei].jpg)](https://assets.ppy.sh/contests/125/winners/Jinairu[Sensei].png)
+
+**2651** voters approved of [**Jinairu[Sensei]**](https://osu.ppy.sh/users/10932731)'s take on all of the official osu! mascots (and Chirou!) lounging around in the springtime grass for a feature shot.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:99]:
```suggestion
**2651** voters approved of [**Jinairu[Sensei]**](https://osu.ppy.sh/users/10932731)'s take on all of the official osu! mascots (and Chirou!) lounging around on the springtime grass for a feature shot.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+**2635** people stopped to marvel at [**Ayamy**](https://osu.ppy.sh/users/10275467)'s take on a blooming pippi undertaking what appears to be a bread run in the midst of a sunlit, springtime day.
+
+Nothing further will be mentioned, especially not the low-hanging fru--damnit.
+
+[![](https://assets.ppy.sh/contests/125/banners/ShovelKun.jpg)](https://assets.ppy.sh/contests/125/winners/ShovelKun.png)
+
+**2406** voters were taken by the hand to a land that they could understand by [**ShovelKun**](https://osu.ppy.sh/users/9948284)'s most adorable take on Mocha out on an evening springtime walk.
+
+An extra special mention goes out to the creative use of theme here - to my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
```
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:113]:
```suggestion
An extra special mention goes out to the creative use of theme here. To my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
```

File: news/2021-05-20-spring-fanart-contest-results.md (hunk 4 above)
Comment (Walavouchey) [inline news/2021-05-20-spring-fanart-contest-results.md:123]:
```suggestion
A huge thank you to everyone who took the time to vote — we know these contests are never easy with the sometimes overwhelming conundrum of choices to be made in the face of so many amazing entries. Another big thank you to every artist who participated too — putting your work out there like this is not easy at all, and we hope you all had fun in the process.
```

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+
+You voted, and the results are in! Check out the winners behind our latest Seasonal backgrounds!
+
+![](https://assets.ppy.sh/contests/125/banners/Tofumang.jpg)
+
+Spring is the season of prosperity and new life, something the artists in the osu! community really took to heart in our latest art contest. They had one not-so-simple task: draw any of our beloved osu! mascots enjoying the park in this season of bloom.
+
+And boy did they deliver.
+
+Over **120** talented creators stepped forth and threw their work into the ring, creating the terrible but wonderful agony for the rest of us that is choosing only **15** of them to vote. But now, the time for pain has passed, and we celebrate the talented finalists as voted for by you.
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2021-05-20-spring-fanart-contest-results.md:15]: These suggestions don't conflict other than whether to write "for voting" or "to vote for". I'm biased when I say that I think my suggestion sounds better. :P

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2021-05-20-spring-fanart-contest-results.md]: address independently

File: news/2021-05-20-spring-fanart-contest-results.md (hunk 2 above)
Comment (Ephemeralis) [reply news/2021-05-20-spring-fanart-contest-results.md:51]: feels better without i think

File: news/2021-05-20-spring-fanart-contest-results.md (hunk 4 above)
Comment (Ephemeralis) [reply news/2021-05-20-spring-fanart-contest-results.md]: either works here and I think I prefer it as is

File: news/2021-05-20-spring-fanart-contest-results.md (hunk 10 above)
Comment (Ephemeralis) [reply news/2021-05-20-spring-fanart-contest-results.md:113]: pause and join with dash/hyphen is much more natural here imo

File: news/2021-05-20-spring-fanart-contest-results.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+
+Here's what you all voted for:
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2021-05-20-spring-fanart-contest-results.md]: addressed this with a reword as well

File: news/2021-05-20-spring-fanart-contest-results.md (hunk 10 above)
Comment (Walavouchey) [reply news/2021-05-20-spring-fanart-contest-results.md:113]: disagree; sentence is too long. when reading, there's a longer pause than what a dash would suggest. a full stop gives gravity to the first sentence.

**Pull request merged**